### PR TITLE
run migrations service

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: .github/workflows/deploy-and-e2e.yml [hunk 1]
```diff
@@ -166,7 +166,6 @@ jobs:
   notify-slack:
     name: Notify Slack on Failure
     runs-on: ubuntu-latest
-    needs: [deploy, discover-tests, test]
```
Comment (cibelius) [inline .github/workflows/deploy-and-e2e.yml:169]: remove needs, so that slack is notified on step failure
